frontend/app/pages/dashboard/dashboard.tsx

```diff
@@ -11,16 +10,24 @@ import {
 } from "~/components/ui/breadcrumb";
 import { useAuthorization } from "~/hooks/use-authorization";
 import { useQuery } from "types/api";
+import { useEffect } from "react";
 
 export function DashboardPage() {
   const { authorizationHeaders } = useAuthorization();
 
-  // Todo: example
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const { data, isLoading } = useQuery("/api/v1/search-profiles", {
+  const {
+    data: profiles,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    isLoading,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    error,
+  } = useQuery("/api/v1/search-profiles", {
     headers: authorizationHeaders,
   });
```

Comment:
```suggestion
  }, { refreshInterval: 0, refreshWhenHidden: false, revalidateOnFocus: false },);
```

less refetches -> less jumping in ui